ci/licenses_golden/licenses_third_party

```diff
@@ -9903,8 +9903,6 @@ ORIGIN: ../../../third_party/dart/benchmarks/DartCLIStartup/dart/DartCLIStartup.
 TYPE: LicenseType.bsd
 FILE: ../../../third_party/dart/benchmarks/DartCLIStartup/dart/DartCLIStartup.dart
 FILE: ../../../third_party/dart/benchmarks/DartCLIStartup/dart2/DartCLIStartup.dart
-FILE: ../../../third_party/dart/benchmarks/IsolateFibonacci/dart/IsolateFibonacci.dart
```

Comment:
Looks like the license files are maybe getting mixed up when rebasing. Maybe you can try backing out all of the changes to them from on your local branch, just take whatever is at HEAD of master, and make the fixes suggested by the tool again.

Reply:
This should do it (remove all changes on the local branch to the license files):

```
git checkout origin/master ci/licenses_golden/
```

Reply:
I'll try this, had some trouble with the tool because I am on mac